tools/WixBuild.wixproj.targets

```diff
@@ -3,41 +3,8 @@
 
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
-  <PropertyGroup>
```

Comment:
No. WixBuild.wixproj.props already has:

```xml
    <SignOutput Condition=" '$(OutputType)'=='Bundle' and '$(PleaseSignOutput)'!='' ">true</SignOutput>
```
Comments in the old WixBuild.wixproj.props indicated that MSIs and CABs wouldn't be signed since the bundle was, so that is likely why. Now, if we want to sign more things (perhaps all the things under /src except, perhaps, DTF), I could centralize it accordingly.

Reply:
:+1: